mapr/metadata.csv

```diff
@@ -0,0 +1,158 @@
+mapr.process.disk_ops.write,count,10,write,,The number of write operations for MapR processes.,0,mapr,
+mapr.process.disk_octets.read,count,10,byte,,The number of bytes read from disk for MapR processes.,0,mapr,
+mapr.process.disk_octets.write,count,10,byte,,The number of bytes written to disk for MapR processes.,0,mapr,
+mapr.process.page_faults.majflt,count,10,error,,The number of major MapR process faults that required loading a memory page from disk.,0,mapr,
+mapr.process.page_faults.minflt,count,10,error,,The number of minor MapR process faults that required loading a memory page from disk.,0,mapr,
+mapr.process.context_switch_voluntary,count,10,process,,The number of voluntary context switches for MapR processes.,0,mapr,
+mapr.process.context_switch_involuntary,count,10,operation,,The number of involuntary context switches for MapR processes.,0,mapr,
+mapr.streams.listen_msgs,count,10,object,,The number of Streams messages read by the consumer.,0,mapr,
+mapr.streams.listen_rpcs,count,10,object,,The number of Streams consumer RPCs.,0,mapr,
+mapr.streams.listen_bytes,count,10,mebibyte,,The number of megabytes consumed by Streams messages.,0,mapr,
+mapr.streams.produce_msgs,count,10,object,,The number of Streams messages produced.,0,mapr,
+mapr.streams.produce_rpcs,count,10,object,,The number of Streams producer RPCs.,0,mapr,
+mapr.streams.produce_bytes,count,10,mebibyte,,The number of megabytes produced by Streams messages.,0,mapr,
+mapr.streams.listen_currpcs,gauge,10,object,,The number of concurrent Stream consumer RPCs.,0,mapr,
+mapr.rpc.bytes_recd,count,10,byte,,The number of bytes received by the MapR Filesystem over RPC.,0,mapr,
+mapr.rpc.bytes_sent,count,10,byte,,The number of bytes sent by the MapR filesystem over RPC.,0,mapr,
+mapr.rpc.calls_recd,count,10,message,,The number of RPC calls received by the MapR filesystem.,0,mapr,
+mapr.topology.utilization,gauge,60,percent,,The aggregate percentage of CPU utilization.,0,mapr,
+mapr.topology.disks_used_capacity,gauge,60,gibibyte,,The amount disk space used in gigabytes.,0,mapr,
+mapr.topology.disks_total_capacity,gauge,60,gibibyte,,The disk capacity in gigabytes.,0,mapr,
+mapr.volume.logical_used,gauge,10,mebibyte,,The number of MBs used for logical volumes before compression is applied to the files.,0,mapr,
+mapr.volume.snapshot_used,gauge,10,mebibyte,,The number of MBs used for snapshots.,0,mapr,
+mapr.volume.total_used,gauge,10,mebibyte,,The number of MB used for volumes and snapshots.,0,mapr,
+mapr.volume.used,gauge,10,mebibyte,,The number of MB used for volumes after compression is applied to the files.,0,mapr,
+mapr.volume.quota,gauge,10,mebibyte,,The number of megabytes(MB) used for volume quota.,0,mapr,
+mapr.volmetrics.read_throughput,gauge,10,kibibyte,second,The per volume read throughput in KB,0,mapr,
+mapr.volmetrics.write_throughput,gauge,10,kibibyte,second,The per volume write throughput in KB,0,mapr,
```

Comment:
How did you determine that the secondary unit is second ?

Reply:
Yeah this one is not 100% sure, because it's a throughput and "second" is the most common time unit I think this is a valid assumption